client/components/Coverages/CoverageEditor/CoverageForm.tsx

```diff
@@ -31,8 +31,7 @@ import {getCoverageFields} from '../../../api/editor/item_planning';
 
 import '../style.scss';
 
-interface IProps {
-    // Values
+interface IOwnProps {
```

Comment:
Pass required arguments instead of making it optional. You can pass `noop` if needed, but that's a good opportunity to document why is it not being passed.